_data/events.json

```diff
@@ -1,338 +1,10825 @@
 [
-   {
-     "name": "AMTA 2024",
-     "id": "amta-2024",
-     "description": "16th biennial conference of the Association for Machine Translation in the Americas",
-     "start_date": "2024-09-30",
-     "end_date": "2024-10-02",
-     "location": {
-       "location":"Chicago, Illinois"
-     },
-     "organizer": {
-       "type": "Organization",
-       "name": "Association for Machine Translation in the Americas",
-       "url": "https://amtaweb.org"
-     },
-     "links": [
-      "https://amtaweb.org/amta-2024/"
-     ],
-     "important_dates": [
+  {
+    "name": "AAMT Seminar",
+    "id": "aamt-seminar",
+    "description": "First Seminar of the Asian-Pacific Association for Machine Translation",
+    "start_date": "2022-09-28",
+    "end_date": "2022-09-28",
+    "more_paragraphs": [
+      "The winners of the 17th AAMT Nagao Award and the 9th AAMT Nagao Award Student Encouragement Award will present their research in the seminar."
+    ],
+    "location": {
+      "online": true
+    },
+    "organizer": {
+      "type": "Organization",
+      "name": "Asian-Pacific Association for Machine Translation",
+      "url": "https://aamt.info/"
```

Comment:
Can we just link to /aamt ?

https://machinetranslate.org/aamt

To avoid repeating this info everywhere.

Reply:
Just use the org slug, and then generate.py can fill it in?
```
"organizer": "aamt"
```